proxy-manager/DOCS.md

```diff
@@ -82,8 +83,16 @@ cleared automatically_
   by taking a snapshot of just the Nginx Proxy Manager add-on, will not
   contain the data from this add-on. Please make sure you backup both
   the Nginx Proxy Manager and the MariaDB add-ons.
-
-## Changelog & Releases
+- As of Home Assitant Core 2021.07 unconfigured reverse proxies are blocked with a `400` http response.  
```

Comment:
- We don't have to keep a changelog of version numbers. Let's remove that.
- `HTTP` is an abbreviation and should be capitalized for that reason.
- Keep line length within 80-chars.

Reply:
can do.

> Keep line length within 80-chars. 
is that all docs or this, [docs guide](https://developers.home-assistant.io/docs/documenting/standards/) said `There is no limit for the line length. You are allowed to write in a flowing text style. This will make it easier to use the GitHub online editor in the future.`?

Reply:
The docs guide linked does not belong to this project and thus does not apply.